pyron-engine/src/main/scala/com/cloudentity/pyron/openapi/OpenApiConverterVerticle.scala

```diff
@@ -159,11 +157,10 @@ class OpenApiConverterVerticle extends ScalaServiceVerticle with OpenApiConverte
     if (exposed.isEmpty) None else Some(buildPath(exposed))
   }
 
-  def isExposed(rules: List[OpenApiRule], targetMethod: HttpMethod, path: String): Boolean = {
-    rules.exists { rule =>
-      val targetPath = rule.targetServicePath
-      val apiGwMethod = rule.rewriteMethod.map(_.value).getOrElse(rule.method)
-      apiGwMethod.toString.toUpperCase == targetMethod.toString.toUpperCase && pathMatches(targetPath, path)
-    }
+  def isExposed(rules: List[OpenApiRule], targetMethod: HttpMethod, path: String): Boolean = rules.exists { rule =>
```

Comment:
yeah, good idea with sharing formatter setting, I think it can be put in the project tree directly. maybe this would be ok? I will move it to next line.